http/user_resource_mapping_service.go

```diff
@@ -361,8 +364,10 @@ func (s *UserResourceMappingService) CreateUserResourceMapping(ctx context.Conte
 	return nil
 }
 
-func (s *UserResourceMappingService) DeleteUserResourceMapping(ctx context.Context, resourceID platform.ID, userID platform.ID) error {
-	url, err := NewURL(s.Addr, memberIDPath(s.BasePath, resourceID, userID))
+// DeleteUserResourceMapping will delete user resource mapping based in criteria.
+func (s *UserResourceMappingService) DeleteUserResourceMapping(ctx context.Context, resourceID influxdb.ID, userID influxdb.ID) error {
+	// default to use org resource type, and member resource type since it doesn't matter.
+	url, err := NewURL(s.Addr, resourceIDUserPath(influxdb.OrgsResourceType, resourceID, influxdb.Member, userID))
```

Comment:
Why are `influxdb.OrgsResourceType` and `influxdb.Member` hardcoded here?

Reply:
we don't really have a delicate URL for removing members. We only have '/orgs/:id/members' or '/checks/:id/members', or whatever. The really path don't seems matter, as they all redirected to the newDeleteMemberHandler.

Reply:
Looks like a bug introduced by https://github.com/influxdata/influxdb/commit/be37e07bea3270751fd28fd9c53ad4353147b29e. The behavior shouldn't be the same regardless of the path.